README.md

```diff
@@ -1,10 +1,11 @@
-﻿## Microsoft Open Source Code of Conduct
+## Microsoft Open Source Code of Conduct
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
 # msftdynamicsgpdocs
-Welcome to the repository for documentation and guides for Microsoft Dynamics GP! The content is available as markdown files (.md), where each file represents an article in the help that publishes to [https://docs.microsoft.com/en-us/dynamics-gp/](https://docs.microsoft.com/en-us/dynamics-gp/). This repo also provides a way for you to actively contribute to the Dynamics GP content, and we welcome your contributions.
+
+Welcome to the repository for documentation and guides for Microsoft Dynamics GP! The content is available as markdown files (.md), where each file represents an article in the help that publishes to [Dynamics GP documentation](https://docs.microsoft.com//dynamics-gp/). This repo also provides a way for you to actively contribute to the Dynamics GP content, and we welcome your contributions.

```

Comment:
Potential issue with the revised link.  It should be:
https://docs.microsoft.com/dynamics-gp/

Reply:
Good eye :-)

Fixed.